LoopKit/CarbKit/CarbStore.swift

```diff
@@ -1432,6 +1446,149 @@ extension CarbStore {
     }
 }
 
+// MARK: Missed / Unannounced Meal Detection
+extension CarbStore {
+    public func hasUnannouncedMeal(insulinCounteractionEffects: [GlucoseEffectVelocity], completion: @escaping (UnannouncedMealStatus) -> Void) {
```

Comment:
I wonder if this could be moved out of CarbStore. Seems like a meal detector would consume data from CarbStore, but otherwise be fairly independent of it.

Reply:
Done! I've moved it into Loop as its own manager (which owns both the meal detection algorithm & notifications)